cwms-data-api/src/main/java/cwms/cda/data/dao/RatingSetDao.java

```diff
@@ -83,11 +89,45 @@ private static String extractOfficeId(String ratingSet) throws JsonProcessingExc
         return office;
     }
 
+    @Override
+    public String retrieveLatestXML(String officeId, String specificationId) {
+        return connectionResult(dsl, c -> {
+            DSLContext context = getDslContext(c, officeId);
+            return CWMS_RATING_PACKAGE.call_RETRIEVE_EFF_RATINGS_XML_F(context.configuration(), specificationId,
+                    Timestamp.from(Instant.now()), Timestamp.from(Instant.now()), null, officeId);
+        });
+    }
+
+    @Override
+    public String retrieveLatestJSON(RatingSet.DatabaseLoadMethod method, String officeId,
```

Comment:
What is the purpose for this method? The `JsonRatingUtils` should be able to convert the XML version to JSON. Doing it this way is retrieving the same rating twice.

Reply:
Removed unnecessary method, added `JsonRatingUtils` conversion